doc/operator-application-capabilities.md

```diff
@@ -558,6 +558,12 @@ spec:
 
 * **NOTE 1**: `httpMethod`, `pattern`, `increment` and `metricMethodRef` fields are required.
 * **NOTE 2**: `metricMethodRef` holds a reference to the existing metric or method map key name `system_name`. In the example, `hits`.
+* **NOTE 3**: The combination of `pattern + httpMethod + metricMethodRef` must be unique among all mapping rules.
+  - If Product CR contains mapping rules where `pattern and httpMethod` are same, even if `metricMethodRef` fields differ -
```

Comment:
this is hard to follow, although we have no rules for what to call a duplicate what not (as of yet) we are saing that pattern method and metric ref must be unique, but then we are saying that when pattern and method are same this is duplication. 
I would much rather have something like, pattern, method and metric must be unique. Having same values across all 3 is a duplicated mapping rule. 
But maybe let's discuss this further when we have duplication pattern established.

Reply:
Thanks very for comment. It's updated, and hope more clear now. If required will be updated after duplication pattern established finally.
Sorry that forgot to create separate commit for review.